.prod/uwsgi.ini

```diff
@@ -1,7 +1,8 @@
 [uwsgi]
 http-socket = :8000
 chdir = /app
-module = tunnistamo.wsgi
+mount = $(APP_URL_PATH)=tunnistamo/wsgi.py
+manage-script-name = true
 static-map = $(MEDIA_URL)=$(MEDIA_ROOT)
 static-map = $(STATIC_URL)=$(STATIC_ROOT)
```

Comment:
How these work together? If there's a `mount` now with a non-empty `$APP_URL_PATH`, are the `static-maps` checked with a path where the `$APP_URL_PATH` has been removed? Or are `static-maps` checked before the `mount`, with the original path?